tracex_project/tracex/views.py

```diff
@@ -1,15 +1,40 @@
 """This file contains the views for the landing page of the tracex app."""
+import os
 from django.views.generic import TemplateView
+from django.shortcuts import render, redirect
+from .forms import ApiKeyForm
```

Comment:
please use absolute paths

Reply:
I dont know why, but I cant figure out why it isnt working with:  

from tracex_project.tracex.forms import ApiKeyForm

I looked up how we imported such things before however we also used . import class and never the absolute path

Reply:
![image](https://github.com/bptlab/TracEX/assets/105175491/bc3c8d46-e38c-484d-82b2-92750b3ece43)

not working

Reply:
![image](https://github.com/bptlab/TracEX/assets/105175491/bf604c09-2b4a-4ec2-a940-5506d3890974)

not working

Reply:
nice! than I will just commit it like that